recipes/virtual_dataframe/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "virtual_dataframe" %}
+{% set version = "v0.5" %}
+
+# For RC, see https://conda-forge.org/docs/maintainer/knowledge_base.html#creating-a-pre-release-build
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pprados/virtual_dataframe/tarball/{{ version }}/virtual_dataframe.tar.gz
+  sha256: 5a5706904ab0505b3838f2bb5ec1a21d961b4d9738ea785de3fa9984294eb058
+
+build:
+  noarch: python
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - build-conda-vdf-env = virtual_dataframe.bin.__init__:main
+
+requirements:
+  host:
+    - python >=3.8
+    - pip >=22.3
+    - setuptools_scm
+
+  run:
+    - python >=3.8
+    - python-dotenv >=0.20
+    - pandas >=1.4
```

```suggestion
    - pandas >1.4
```
Ref: https://github.com/pprados/virtual_dataframe/blob/616cf6c3ad8105ac706ed46c1ecb6a1b978517c4/setup.py#L13

---

Why? I need to be compatible with panda 1.4 and 1.5. `>=1.4` is correct for me.
`pandas >1.3` ?